sdk/eventhub/event-hubs/src/eventPosition.ts

```diff
@@ -33,13 +33,14 @@ export interface EventPositionOptions {
 }
 
 /**
- * Represents options can be set during the creation of a event hub receiver.
- * Defines a position of an @link~EventData in the event hub partition.
- * @class EventPosition
+ * The position of events in an Event Hub partition, typically used in the creation of
+ * an `EventHubProducer`.
+ * @class
  */
 export class EventPosition {
   /**
-   * @property The offset from which events would be received: `"-1"`.
+   * @property The token that represents the beginning event in the stream of a partition.
+   * Defaults to `"-1"`.
```

Comment:
`Defaults to` implies that something can hold a non default value. That isnt the case here.
Also, why isnt `endOfStream` updated that is right after the startOfStream here?

Reply:
Updated this and `endOfStream`. Both are actually private members and won't be present in the docs.